tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodMetadataInfo.cs

```diff
@@ -27,6 +36,20 @@ public MethodMetadataInfo(string[] parameterNames, string[] localVariableNames,
         /// </summary>
         public string[] LocalVariableNames { get; }
 
+        /// <summary>
+        /// Gets the names and the values of the async method's local variable from the hoist object - i.e. the state machine.
```

Comment:
```suggestion
        /// Gets the names and the values of the async method's local variable from the heap-allocated object that holds them - i.e. the state machine.
```

Reply:
Fixed in 871a652d9b150d63443d87a542e34d4643145adf